src/NzbDrone.Core/Notifications/MediaBrowser/MediaBrowserProxy.cs

```diff
@@ -31,11 +31,23 @@ public void Notify(MediaBrowserSettings settings, string title, string message)
             ProcessRequest(request, settings);
         }
 
-        public void Update(MediaBrowserSettings settings, int tvdbId)
+        public void Update(MediaBrowserSettings settings, string seriesPath, string updateType)
         {
-            var path = string.Format("/Library/Series/Updated?tvdbid={0}", tvdbId);            
+            var path = "/Library/Media/Updated";
             var request = BuildRequest(path, settings);
-            request.Headers.Add("Content-Length", "0");
+            request.Headers.ContentType = "application/json";
+
+            request.SetContent(new
+            {
+                Updates = new[]
+                {
+                    new
+                    {
+                        Path = seriesPath,
```

Comment:
IIRC there was mention of path substitution on the Emby side, but that GHI/PR is gone with the privatization of their repo. That really was the entire point of fetching the current path instead of passing Sonarr's path, but unfortunately that was ripped out.

I don't think it makes sense for Sonarr have another path substitution specifically for Emby, but we can reconsider later on.